kubejs/server_scripts/mods/modern_industrialization/blast_furnace.js

```diff
@@ -39,6 +40,28 @@ ServerEvents.recipes(e => {
         e.custom(newRecipe).id(id);
     }
 
+    // -- NETHER STAR -> MOLTEN NETHER STAR -- //
+    blastFurnace(
+        st('nether_star'),
+        128,
+        300,
+        [ { amount: 1, item: mc('nether_star') } ],
+        null,
+        null,
+        [ { amount: 500, fluid: mi('molten_nether_star') } ]
+    );
+
+    // -- IMPURRE LIQUID NETHER STAR -> MOLTEN NETHER STAR -- //
+    blastFurnace(
+        st('molten_nether_star'),
+        128,
+        900,
+        [ { amount: 1, item: mi('platinum_dust'), probability: 0.05 } ],
```

Comment:
Change to `mi('platinum_tiny_dust')` and remove the probability

Reply:
Should it consume the platinum dust or should it not consume the dust?

Reply:
Yes, it should consume it